### PR TITLE
easier readability and links

### PR DESCRIPTION
**Pull request by xamionex**
title says it all

### CODE REVIEW

**Comment by xamionex** [comment]
I need to add links on SUMMARY
please wait until i do so <3

**Comment by xamionex** [comment]
[web-checkout](https://amione.gitbook.io/)

File: docs/installing-northstar/troubleshooting.md [hunk 1]
```diff
@@ -4,74 +4,72 @@ Generally try to first launch the vanilla game (i.e. not Northstar) if you encou
 
 A lot of problems around the game failing to communicate with Origin can also be prevented by launching Origin before Northstar should you encounter any issues in that regard.
 
-Also note that some solutions described below can also apply to the base game, like issues caused by [10th+ gen Intel CPUs](#cant-join-servers-issues-with-10th+-gen-intel-cpus).
+Also note that some solutions described below can also apply to the base game, like issues caused by [10th+ gen Intel CPUs](troubleshooting.md#cant-join-servers).
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: That new ref doesn't seem to work for me compared to the old one

File: docs/installing-northstar/troubleshooting.md [hunk 2]
```diff
@@ -4,74 +4,72 @@ Generally try to first launch the vanilla game (i.e. not Northstar) if you encou
...
 This error is usually caused by running the `NorthstarLauncher.exe` in the wrong location, such as extracting the zip it came with directly in your Downloads folder and running it there.
 
-- Default Steam Location: `%ProgramFiles(x86)%\Steam\steamapps\common\Titanfall2\`
-- Default Origin Location: `%ProgramFiles(x86)%\Origin Games\Titanfall2\`
+* Default Steam Location: `%ProgramFiles(x86)%\Steam\steamapps\common\Titanfall2\`
+* Default Origin Location: `%ProgramFiles(x86)%\Origin Games\Titanfall2\`
 
-**If it still appears after trying the fix above**
-It's possible that you may have **corrupted or missing** game files
+**If it still appears after trying the fix above** It's possible that you may have **corrupted or missing** game files
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
**If it still appears after trying the fix above:** It's possible that you may have **corrupted or missing** game files
```

File: docs/installing-northstar/troubleshooting.md [hunk 3]
```diff
@@ -4,74 +4,72 @@ Generally try to first launch the vanilla game (i.e. not Northstar) if you encou
...
 
-- **Solution 1**: Launch EA Desktop and the game as admin once so that it can copy that file.
-- **Solution 2**: If you know how, just copy the mentioned file manually, remembering you need to change its filename (just use some temp dir and rename there).
-- **Solution 3**: If you know how, just change the folder permissions in Properties->Security tab of your Titanfall2 install dir to let your user write.
+* **Solution 1**: Launch EA Desktop and the game as admin once so that it can copy that file.
+* **Solution 2**: If you know how, just copy the mentioned file manually, remembering you need to change its filename (just use some temp dir and rename there).
+* **Solution 3**: If you know how, just change the folder permissions in Properties->Security tab of your Titanfall2 install dir to let your user write.
 
-## Can't Join Servers (Issues with 10th+ gen Intel CPUs)
+## Can't Join Servers <a href="#intel" id="intel"></a>
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: I'd keep the `(Issues with 10th+ gen Intel CPUs)`. There can be a multitude of reasons why someone isn't able to join a specific or any servers and keeping the reference to the CPU in the heading should help prevent someone with unrelated issues from trying this solution. ^^

File: docs/installing-northstar/troubleshooting.md [hunk 4]
```diff
@@ -90,68 +88,68 @@ If you're on Linux, you can set the appropriate environment variable via `env OP
...
 
-## I can't play the Campaign
+## I can't play the Campaign <a href="#campaign" id="campaign"></a>
 
-Campaign is not supported in 1.4, but you could do one of these **"_fixes_"**
+Campaign is not supported in 1.4, but you could do one of these **"**_**fixes**_**"**
 
 1. Doing `ns_auth_allow_insecure 1` in the console
-    - [If you can't open the console](#i-cant-open-the-console)
+   * [If you can't open the console](troubleshooting.md#i-cant-open-the-console)
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: The change broke the ref, so clicking the link no longer works. I'd just drop the `troubleshooting.md` again

```suggestion
   * [If you can't open the console](#i-cant-open-the-console)
```

File: docs/installing-northstar/troubleshooting.md [hunk 5]
```diff
@@ -90,68 +88,68 @@ If you're on Linux, you can set the appropriate environment variable via `env OP
...
 2. Adding `-vanilla` to your launch options
-    - [How to add launch options](#adding-launch-options)
+   * [How to add launch options](troubleshooting.md#adding-launch-options)
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
   * [How to add launch options](#adding-launch-options)
```

ditto

File: docs/installing-northstar/troubleshooting.md [hunk 6]
```diff
@@ -90,68 +88,68 @@ If you're on Linux, you can set the appropriate environment variable via `env OP
...
 
-## Authentication Failed when clicking Launch Northstar
+## Authentication Failed when clicking Launch Northstar <a href="#lsx2" id="lsx2"></a>
 
-Before trying this check out [this section](#failed-to-initialize-origin-origin-core-seems-to-be-running-but-the-lsx-authentication-challenge-fai).\
+Before trying this check out [this section](troubleshooting.md#lsx-authentication-failed).\
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: Also broken. On a sidenote, Gitbook doesn't seem to be a fan of `<a>` style refs :I

File: docs/installing-northstar/troubleshooting.md [hunk 7]
```diff
@@ -90,68 +88,68 @@ If you're on Linux, you can set the appropriate environment variable via `env OP
...
 If that didn't work DM `Petar#0389` on Discord and tell him if it did or did not work.\
 Alternative to that fix:
 
 1. Close the game
 2. Open task manager
 3. End Origin (everything origin related)
 4. Launch Origin as admin
-5. Start the game through Origin with `-northstar` in [launch options](#adding-launch-options)
+5. Start the game through Origin with `-northstar` in [launch options](troubleshooting.md#adding-launch-options)
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
5. Start the game through Origin with `-northstar` in [launch options](#adding-launch-options)
```

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]
Aight, looks like gitbook doesn't like the changed ref format. Maybe revert those that are now broken. I think I annotated them all but might wanna double check, just in case ^^

**Comment by xamionex** [comment]
Lol, gitbook things not working when gitbook created em.

**Comment by xamionex** [comment]
If it still doesn't work on the [web-checkout](https://amione.gitbook.io/) just hit CTRL + F5 to refresh without cache.

**Review by GeckoEidechse — APPROVED** [review]

**Pull request merged**